index.js

```diff
@@ -177,15 +238,22 @@ var PiwikTracker = function(opts) {
 			push(['enableLinkTracking']);
 		}
 
-		if (opts.injectScript) {
-			var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript'; g.defer=true; g.async=true; g.src=u+opts.clientTrackerName;
+		if (opts.injectScript && !alreadyInitialized) {
+			var d=document;
+			var g=d.createElement('script');
+			var s=d.getElementsByTagName('script')[0];
+			g.type='text/javascript';
+			g.defer=true;
+			g.async=true;
+			g.src=u+opts.clientTrackerName;
 			s.parentNode.insertBefore(g,s);
+			g.setAttribute('data-' + opts.piwikScriptDataAttribute, opts.siteId);
 		}
 	})();
 
 	// return api
 	return {
-    _isShim: false,
+		_isShim: false,
 		track: track,
 		push: push,
 		setUserId, setUserId,
```

Comment:
This should probably be `setUserId: setUserId` (colon, not comma)

Reply:
oh, how could that pass the tests 🤔